client/src/components/shared/Helpfulness.jsx

```diff
@@ -47,6 +47,23 @@ class Helpfulness extends React.Component {
     }
   }
 
+  componentDidUpdate(prevState) {
+    const target = Object.keys(this.props)[0];
+    const targetObj = this.props[target];
+
+    if (target === 'review') {
+      const targetId = targetObj['review_id'];
+      if (this.state.targetId !== targetId) {
+
+        this.setState({
+          helpfulCount: targetObj['helpfulness'],
+          targetId: targetId,
+          link: '/reviews/' + targetId + '/helpful'
+        });
+      }
+    }
+  }
```

Comment:
Hi Alex , just wonder why we need componentDidUpdate ? I have tested it and it work fine, are there any bugs ?

Reply:
When I'm using Sort button to sort my reviews, the helpfulness doesn't update. Let me know if you have a minute to test it out, maybe I'm missing something.

Reply:
Per our chat, keeping ComponentDidUpdate for now, for Sort filters